src/Chainweb/Pact/InternalTypes.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE StrictData #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+-- |
+-- Module: Chainweb.Pact.InternalTypes
+-- Copyright: Copyright © 2019 Kadena LLC.
+-- License: See LICENSE file
+-- Maintainer: Mark Nichols <mark@kadena.io>
+-- Stability: experimental
+--
+-- Pact Types module for Chainweb
```

Comment:
What is the rational for breaking this out; if justified then update the module header accordingly here and record yourself as maintainer

Reply:
I moved everything that was here to a different location instead